### ISSUE TITLE
Every `addtimer()` throws an exception

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#259]
`addtimer(CALLBACK(src, .proc/clear_fullscreen_after_animate, screen), animated, TIMER_CLIENT_TIME)`
```
#define GLOBAL_PROC    "some_magic_bullshit"
/// A shorthand for the callback datum, [documented here](datum/callback.html)
#define CALLBACK new /datum/callback
```
![image](https://user-images.githubusercontent.com/5714543/127758977-c0b1cd72-8cf8-467d-8d99-2c29865b6872.png)

BeeStation

**Comment by ike709** [comment]
Preprocessor output: `_addtimer(new /datum/callback(src, .proc/clear_fullscreen_after_animate, screen),animated,(1<<2),, file = "code\_onclick\hud\fullscreen.dm", line = 32)`

**Comment by ike709** [comment]
Seems to be an issue with `args`: `#define addtimer(args...) _addtimer(args, file = __FILE__, line = __LINE__)`

**Comment by ike709** [comment]
Seems to be `__file__` not escaping slashes in paths

**Comment by wixoaGit** [comment]
Fixed by 610f69ea209ea8b30613303decaab5b82d1f0f8b.
